docs/learn/dapp-staking-v3/index.md

```diff
@@ -0,0 +1,62 @@
+
+dApp staking on the Astar/Shiden Network operates similarly to staking on validators. However, in this new paradigm, individuals known as dApp stakers or nominators have the ability to nominate their Astar/Shiden tokens on specific dApps they wish to support.
+
+At every block on the network, a portion of the rewards is allocated to dApp staking. This reward is then distributed between the operators (developers) of the dApps and the nominators. This mechanism serves as a powerful incentive for developers to build and deploy their dApps on the Astar/Shiden Network.
+
+
+## Benefits of dApp Staking on Astar/Shiden
+
+For Web3 to flourish, there needs to be a symbiotic relationship between dApp stakers (i.e. nominators), dApp developers, and dApp users. On Astar/Shiden, dApp staking is the mechanism that enables this mutually beneficial relationship to occur.
+
+- Developer Compensation: Unlike traditional approaches where dApp developers need to rely on grant programs, token issuance, and fundraising efforts to generate income, dApp staking on Astar/Shiden provides developers with a basic income. As long as a dApp has been nominated, developers can receive a steady stream of rewards.
+
+- Increased Popularity, Increased Reward: As a dApp gains popularity and attracts more nominators from the community, the developers who built the dApp can receive a larger percentage of the block rewards. This incentivizes developers to create high-quality dApps that resonate with users and foster community engagement.
+
+- Reduced Dependency on Gas Fees: Building on other blockchains often entails paying substantial gas fees for transactions and interactions. However, on the Astar/Shiden Network, developers can receive compensation through dApp staking, minimizing their reliance on gas fees and making it more economically feasible to continue dApp development.
+
+### Benefits for Stakers (Nominators): Increase in Token Value & High APRs
+
+Astar/Shiden dApp stakers (i.e. nominators) want the value of their tokens to increase but for the value of their tokens to increase, the underlying value of the asset — the Astar/Shiden Network — needs to increase.
+
+Another way for the value of tokens to increase is if the total circulation of tokens decreases. The more tokens are staked, the fewer tokens are in circulation, and the fewer tokens are in circulation, the higher the price.
+
+This market dynamic enables dApp stakers to get more long-term value from their tokens while earning high APRs from staking their tokens to dApps.
+
+Please refer to [this section](/docs/build/dapp-staking/for-stakers/) to learn more about participating as a staker.
+
+### Benefits for Builders: Build2Earn
```

Comment:
Should `Build2Earn` be replaced with `Rewards` or `Earn` given how the naming discussion went?

Reply:
I am not sure if this is resolved fully but I will use just Earn Rewards then till something else changes. thx.